### PR TITLE
Matching to design

### PR DESCRIPTION
**Pull request by DorotaKar**

### CODE REVIEW

**Review by maxFadet — APPROVED** [review]

File: src/common/Rates/index.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ export const Rates = ({ mainInfo, hideTotalScore, voteAverage, voteCount }) => {
                                 {Number(voteAverage).toFixed(1).replace('.', ',')}
                                 <TotalScore $hideTotalScore={isTotalScoreHidden} $larger={isRatesAreMainInfo}>/ 10</TotalScore>
                             </Score>
-                            <Votes $larger={isRatesAreMainInfo}>{Number(voteCount)} votes</Votes>
+                            <Votes $larger={isRatesAreMainInfo}>
+                                {Number(voteCount).toLocaleString().replace(/,/g, ' ')} votes</Votes>
```
Comment (OskarWlaszczuk) [inline src/common/Rates/index.js:26]: Hey, that is good idea to sepearte numbers, with spaces but in this case design requires a comma.
Here my suggestion:

- **Number(voteCount).toLocaleString().replace(/,/g, ' ,')**
This code replace comma in vote with space on the left and comma.
In my opinion it more matches the design

File: src/common/Rates/styled.js [hunk 2]
```diff
@@ -96,9 +96,9 @@ export const TotalScore = styled.span`
 `;
 
 export const Votes = styled.p`
-  color: ${({ theme }) => theme.colors.black};
+  color: ${({ theme }) => theme.colors.waterloo};
   margin: 0;
-  font-size: 14px;
+  font-size: 16px;
```
Comment (OskarWlaszczuk) [inline src/common/Rates/styled.js:101]: I would not change none of this styles in this file, they already matches the design.
Default styles of all styled-components in this file, refers to components like Tile, or DetailsTile where the sizes of elements are smaller. If some of the styled-components gets $larger attribude, its increase their sizes or change layout to match MovieBanner component design, where all styles in Rates component are larger.

**Review by OskarWlaszczuk — CHANGES_REQUESTED** [review]

File: src/common/Rates/index.js (hunk 1 above)
Comment (DorotaKar) [reply src/common/Rates/index.js:26]: In the design it doesn't show exactly what the number of votes above 1000 should look like - it seems to me that the version with a space is more universal for Europe - but globally the version with a comma will actually be better :ok_hand:

File: src/common/Rates/styled.js (hunk 2 above)
Comment (DorotaKar) [reply src/common/Rates/styled.js:101]: the project Tile component on MovieListPage scores and votes have the same font size (on large screens 16px, on small 13px). Reducing the font on one of them is visible and the effect is different from the project.
maybe let's create a separate styled component for tile in movie details?

**Pull request merged**